src/scmdata/run.py

```diff
@@ -1553,13 +1559,28 @@ def process_over(
             This functionality is disabled if na_override is None, but may result in incorrect
             results if the timeseries meta includes any nan's.
 
+        op_cols: dict of str: str
+            Dictionary containing any columns that should be overridden after processing.
+
+            If a required column from :class:`scmdata.ScmRun` is specified in ``cols`` and
+            ``as_run=True``, an override must be provided for that column in ``op_cols``
+            otherwise the conversion to :class:`scmdata.ScmRun` will fail.
+
+        as_run: bool or subclass of BaseScmRun
```

Comment:
```suggestion
        as_run: bool or class
```

? Can be any class now right?

Reply:
Ignore me just saw the code